src/js/auction/bidbox/components/ActionButton.js

```diff
@@ -4,7 +4,7 @@ export default function ActionButton(props) {
   return (
     <button
       {...props}
-      className={`button action-button is-rounded is-background-gradient-dark has-text-white {props.className}`}
+      className={`button action-button is-rounded is-background-gradient-dark has-text-white is-button-styled {props.className}`}
```

Comment:
I think it would have been better to just override `button`. This way we get it automatically for every button. But can be refactored.